PayGreen/paygreen-ui
# PGX-4421 - Nouveau composant Badge

## How to test ?
- Launch the storybook with yarn storybook
- Go to section Badge
- Check that the Badge respond correctly to the differents options
